ImperatorToCk3/Source/Main.cpp

```diff
@@ -0,0 +1,9 @@
+#include "ImperatorToCk3Converter.h"
+
+
+int main()
```

Comment:
Updates to add soon:
1. The (converter) version functionality from EU4 to Vic2 and Vic2 to HoI4.
2. A try/catch block to grab any exceptions and log the message, then return -1.

Reply:
+1.
The version is important as it's often the only way to debug people's logs. Try/catch is also necessary so we may see some of the errors. They won't be output if the converter crashes outright, like when you attempt to read the name of a character's family, without first setting family pointer.

Reply:
I think all's done now.